crates/solver/src/metrics.rs

```diff
@@ -132,15 +132,8 @@ impl Metrics {
                 "Surplus ratio differences between winning and best settlement per order",
             )
             .buckets(vec![
-                -f64::INFINITY,
-                -0.1,
-                -0.01,
-                -0.005,
-                0.,
-                0.005,
-                0.01,
-                0.1,
-                f64::INFINITY,
+                -2.0, // Difference of percentages can not exceed +/- 2
```

Comment:
Why not? Isn't the difference between 0 and 100% more than two? I guess I don't really understand what exactly we are measuring here.